src/Common/ProfileEvents.cpp

```diff
@@ -8,6 +8,7 @@
 /// If the event is generic (i.e. not server specific)
 /// it should be also added to src/Coordination/KeeperConstant.cpp
 #define APPLY_FOR_BUILTIN_EVENTS(M) \
+    M(MemoryCredits, "Memory usage credits accumulated over time, in microseconds*bytes") \
```

Comment:
Arent microseconds*bytes can be really big number (plus, with high randomity in it)?
May be, it make sense to have it as microseconds*MB or round lower bits, but keep bytes then.

Reply:
No, let's keep bytes.